geomesa-accumulo/geomesa-accumulo-distributed-runtime/src/main/resources/install-geomesa-namespace.sh

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+# Installs a GeoMesa distributed runtime JAR into an Accumulo namespace
+
+while getopts ":u:p:n:g:h:" opt; do
+  case $opt in
+    u)
+      ACCUMULO_USER=$OPTARG
+      ;;
+    p)
+      ACCUMULO_PASSWORD=$OPTARG
+      ;;
+    n)
+      ACCUMULO_NAMESPACE=$OPTARG
+      ;;
+    g)
+      GEOMESA_JAR=$OPTARG
+      ;;
+    h)
+      HDFS_URI=$OPTARG
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Option -$OPTARG requires an argument." >&2
+      exit 1
+      ;;
+  esac
+done
+
+if [ -z "$ACCUMULO_USER" ]; then
+    echo "Accumulo username parameter is required: -u" >&2
+    ERROR=1
+fi
+
+if [ -z "$ACCUMULO_NAMESPACE" ]; then
+    echo "Accumulo namespace parameter is required: -n" >&2
+    ERROR=1
+fi
+
+if [ -z "$GEOMESA_JAR" ]; then
+    # cd to directory of script and look for GeoMesa JAR
+    cd "$( dirname "${BASH_SOURCE[0]}" )"
+    GEOMESA_JAR=$(ls | grep geomesa-accumulo-distributed-runtime)
+    if [ -z "$GEOMESA_JAR" ]; then
+        echo "Null GeoMesa JAR parameter encountered and could not find a GeoMesa distributed runtime JAR"
+        ERROR=1
+    else
+        echo "Null GeoMesa JAR parameter encountered, using $GEOMESA_JAR"
+    fi
+fi
+
+if [[ -z "$HDFS_URI" ]]; then
```

Comment:
[picky] This is the second syntax for a conditional statement; any reason for the inconsistency?